### ISSUE TITLE
r_lodCurveError

### ISSUE THREAD

**Issue by WhiteMagicRaven** [zturtleman/lilium-voyager#12]
this cvar is cheat protected, actually this is not cheat cvar.
Its control level of detail, higher values removes LODS making them more beautiful. 
Please allow to change this value to higher values but not less then 250, Thanks

Makes graphic better

**Comment by zturtleman** [comment]
Yeah, I tend to agree with this but there is no consensus upstream. https://github.com/ioquake/ioq3/issues/383
